test/parallel/test-fs-utimes.js

```diff
@@ -160,13 +160,15 @@ function runTests(iter) {
 const path = `${tmpdir.path}/test-utimes-precision`;
 fs.writeFileSync(path, '');
 
-// Test Y2K38 for all platforms [except 'arm', 'OpenBSD', 'SunOS' and 'IBMi']
-if (!process.arch.includes('arm') &&
-  !common.isOpenBSD && !common.isSunOS && !common.isIBMi) {
+{
   const Y2K38_mtime = 2 ** 31;
   fs.utimesSync(path, Y2K38_mtime, Y2K38_mtime);
   const Y2K38_stats = fs.statSync(path);
-  assert.strictEqual(Y2K38_stats.mtime.getTime() / 1000, Y2K38_mtime);
+  const mtimeStamp = Y2K38_stats.mtime.getTime() / 1000;
+  // Off-by-one value is permissible for file systems that don't support Y2K38.
```

Comment:
I think skipping as was done before it better. The sets it to off by one, so allowing off by one is the same thing.

Reply:
The previous skipped platforms are a separate issue that I was optimistically hoping this change would fix. If we dont allow the max - 1 value, then we need to skip all Linux hosts.

I think the thing to do is restore the previous skipped platforms *and* allow max - 1.